ui/src/index.css

```diff
@@ -25,21 +38,6 @@ a:hover {
   color: #535bf2;
 }
 
-body {
-  margin: 0;
-  display: block; /* Change flex to block to remove centering */
-  min-width: 320px;
```

Comment:
Probably not many mobile devices around these days with widths below 320px, but still not something to be assuming.